rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -238,7 +238,7 @@ def testQuery_noParticipants(self):
 
   def submit_questionnaire_response(self, participant_id, questionnaire_id, race_code, gender_code,
                                     first_name, middle_name, last_name, zip_code,
-                                    state_code, street_address, city, sex_code,
+                                    state_code, street_address, city, sex_code, login_phone_number,
```

Comment:
Please add tests in here that confirm:

1) If you pass up a login phone number in a questionnaire response, it comes back in the participant summary.
2) You can query on loginPhoneNumber and get back matching participant(s)

You can either do this in a new test method or pile on to an existing one like this:

https://github.com/all-of-us/raw-data-repository/blob/3bd458de95d02a3a2bf802089453975df0eca016/rest-api/test/unit_test/api_test/participant_summary_api_test.py#L757

Let me know if you have questions about how to do this.... I know the code is a bit of a mess. :)

Reply:
Hi Dan, I add this to the existing method, please review.